### ISSUE TITLE
Version 2.9.11 includes PHP syntax incompatible with PHP 5.6 (yet, package set to Requires PHP: 5.6)

### ISSUE THREAD

**Issue by quinncomendant** [pods-framework/pods#6988]
### Description

Pods automatically updated to 2.9.11 on a legacy site running on PHP 5.6. The site crashed with this error:

```
PHP Parse error:  syntax error, unexpected ':', expecting '{' in /home/u/example.com/html/wp-content/plugins/pods/tribe-common/src/functions/files.php on line 19
```

The `tec_is_file_from_plugins` function from this file uses syntax that requires at least PHP 7.4:

- [Spread operator](https://wiki.php.net/rfc/spread_operator_for_array): `string ...$plugin_files`
- [Function return type declaration](https://www.php.net/manual/en/language.types.declarations.php): `): bool {`

You will need to either refactor this code to support PHP 5.6, or update the minimum PHP version required for running Pods to PHP 7.4.

**I did not test if there is code elsewhere that does not support PHP 5.6.** I resolved the issue by downgrading to Pods 2.9.10.

**Comment by sc0ttkclark** [comment]
We backtracked the tribe-common library shipped in Pods shortly after Pods 2.9.11 was released to deal with issues like these. Closing since that's now resolved.

**Comment by quinncomendant** [comment]
So, do you mean you refactored to support PHP 5.6, or updated the required PHP version to PHP 7.4?
